### PR TITLE
WIP: tenant dashboards

### PR DESCRIPTION
**Pull request by xaun**
@ouranos just FYI, any comments?

### CODE REVIEW

**Comment by xaun** [comment]
FYI, waiting for https://github.com/maestrano/mno-enterprise/pull/535 to be approved & merged before adding specs for this.

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module MnoEnterprise::Concerns::Controllers::Jpi::V1::Impac::TenantDashboardsController
```
Comment (cesar-tonnoir) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb]: Shouldn't it be an admin controller? I thought this was used to load the admin dashboards in your new section in the admin panel?

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    dashboards
+    render template: 'mno_enterprise/jpi/v1/impac/dashboards/index'
+  end
+
+  private
+
+  def dashboards
+    @dashboards ||= MnoEnterprise::Dashboard
+      .includes(*DASHBOARD_DEPENDENCIES)
+      .find(owner_type: 'Mnoe::Tenant')
```
Comment (cesar-tonnoir) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb]: When is the dashboard created with `owner_type = 'Mnoe::Tenant'`? Is it directly in mnohub?

Comment (xaun) [reply api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb]: As you now know, currently it is not being created, as I got this particular element of the project confused with the SPA and thought that we were also provisioning a dashboard for the tenant.

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb (hunk 1 above)
Comment (xaun) [reply api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb]: Yes true, this should be in the admin controller

**Comment by xaun** [comment]
Replace by: https://github.com/maestrano/mno-enterprise/pull/557

**Pull request closed without merging**